scripts/circle-setup

```diff
@@ -33,7 +33,7 @@ prepare_system() {
     sudo sysctl -w net.ipv4.conf.all.route_localnet=1
     sudo sysctl -w net.bridge.bridge-nf-call-iptables=1
     sudo sysctl -w net.ipv4.ip_forward=1
-    sudo iptables -t nat -I POSTROUTING -s 127.0.0.1 ! -d 127.0.0.1 -j MASQUERADE
+    #   sudo iptables -t nat -I POSTROUTING -s 127.0.0.1 ! -d 127.0.0.1 -j MASQUERADE
```

Comment:
For the final submit, can you remove these commented lines or add a comment as to why they're commented out?

Reply:
This is a temporary test commit, I think I marked it as such. Some of this is breaking networking, trying to figure out what as I can't repro this locally.